mix.exs

```diff
@@ -99,6 +99,7 @@ defmodule Livebook.MixProject do
       {:aws_signature, "~> 0.3.0"},
       {:mint_web_socket, "~> 1.0.0"},
       {:protobuf, "~> 0.8.0"},
+      {:erlfmt, "~> 1.0"},
```

Comment:
We can send it across (like you are doing already for a few of the necessary runtime packages).

Reply:
We only sent across a few internal modules that we need for evaluation, but I think sending all modules from an external package is too brittle (especially that theoretically it can rely on other applications).

@josevalim we already hide the option, but that option is only for persisting the notebook. For persisting we could actually format using `:erlfmt`, since we do that in the Livebook node, but having one without the other may be weird?

Reply:
Agreed that one without the other is weird. I am fine with dropping formatting for now.